pkg/redis/redis_client.go

```diff
@@ -27,6 +27,7 @@ type Client interface {
 	Expire(key string, ttl time.Duration) (bool, error)
 	FlushDB() (string, error)
 	Set(string, interface{}, time.Duration) error
+	SetNX(string, interface{}, time.Duration) (bool, error)
```

Comment:
we decided that we want to have parameter names in interfaces, even though we did not yet add them everywhere. Please adhere to this rule when you add code. Maybe you could also quickly add the missing parameter names in the other functions?

Reply:
Should be much better now.